README.md

```diff
@@ -115,7 +115,32 @@ In order to execute CDS logic, the FHIR Clinical Guidelines Implementation guide
 
 ### Run CQL Tests
 
-To run the 1000+ tests to verify functionality of the CCSM CDS, type `npm run test-cql` into a command prompt from within the same directory as this README file. Note that this will require a UMLS account and VSAC API key; please see documentation for the [CQL Testing Framework](https://github.com/AHRQ-CDS/CQL-Testing-Framework) for further instructions. The results from running the tests against the current CQL definitions can be found in [test-results.txt](./test-results.txt).
+To run the 1000+ tests to verify functionality of the CCSM CDS, type `npm run test-cql` into a command prompt from within the same directory as this README file. Note that this will require a UMLS account and VSAC API key; please see the section below, [Setting Up VSAC](#setting-up-vsac), for further instructions. The results from running the tests against the current CQL definitions can be found in [test-results.txt](./test-results.txt).
```

Comment:
I think the second paragraph of the "[Downloading Value Sets](https://github.com/AHRQ-CDS/CQL-Testing-Framework?tab=readme-ov-file#downloading-value-sets)" section of the CQL Testing Framework README provides some useful information, especially around where the value sets get stored, and the fact that you only need to download value sets before the first run and thatn they will be cached afterwards. Can we add something similar to our "Run CQL Tests" section?